modules/db/src/main/scala/gem/dao/ObservationDao.scala

```diff
@@ -21,6 +21,7 @@ object ObservationDao {
                     ${o.id.index},
                     ${o.title},
                     ${o.instrument})
+       ON CONFLICT DO NOTHING
```

Comment:
The reason I'm avoiding "check first and then insert" is that it's a race condition unless you have your transaction isolation set to `SERIALIZABLE` which is very expensive. We can however turn it on for just this one interaction if it ends up being hard to manage pessimistically.